src/transport/raw/tests/TestTCP.cpp

```diff
@@ -399,110 +391,60 @@ void chip::Transport::TCPTest::CheckProcessReceivedBuffer(nlTestSuite * inSuite,
 
     Transport::PeerAddress lPeerAddress    = Transport::PeerAddress::TCP(addr);
     TCPBase::ActiveConnectionState * state = tcp.FindActiveConnection(lPeerAddress);
-    NL_TEST_ASSERT(inSuite, state != nullptr);
+    EXPECT_NE(state, nullptr);
```

Comment:
```suggestion
    ASSERT_NE(state, nullptr);
```

In all places where something is checked for being not-null, check what happens next. If such pointer is dereferenced, then use ASSERT instead of EXPECT.

Reply:
Changed.